lib/public/utilities/fetch/buildUrl.js

```diff
@@ -98,6 +99,7 @@ export const buildUrl = (baseURL, parameters) => {
      * @return {void}
      */
     const formatAndPushQueryParameter = (key, value) => {
+        key = key.replace(/=/g, '%3D');
```

Comment:
Why this?

Reply:
to use '=' as numeric operator and as part of nested keys